RDKShell/RDKShell.cpp

```diff
@@ -397,6 +442,34 @@ namespace WPEFramework {
             }*/
         }
 
+        static bool isClientExists(std::string client)
+        {
+            bool exist = false;
+            for (int i=0; i<gCreateDisplayRequests.size(); i++)
```

Comment:
Let's add a mutex around `gCreateDisplayRequests` to ensure this list is not modified at the same time by another thread.